app/src/main/java/com/iitdh/sonusourav/instigo/Utils/CommonFunctions.java

```diff
@@ -86,6 +92,18 @@ public static boolean navigationItemSelect(MenuItem item, final Activity activit
                 FirebaseAuth loginAuth=FirebaseAuth.getInstance();
                 loginAuth.signOut();
 
+                GoogleSignInOptions gso = new GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
+                        .requestIdToken(activity.getResources().getString(R.string.default_web_client_id))
+                        .requestEmail()
+                        .build();
+                GoogleSignInClient googleSignInClient = GoogleSignIn.getClient(activity, gso);
+                googleSignInClient.signOut().addOnCompleteListener(new OnCompleteListener<Void>() {
+                    @Override
+                    public void onComplete(@NonNull Task<Void> task) {
+
+                    }
+                });
+                
                 Toast.makeText(activity.getApplicationContext(), "User successfully logged out.", Toast.LENGTH_SHORT).show();
```

Comment:
How about moving the Toast and intent inside onComplete()? That way the addition will make more sense otherwise, chances are there that there might be an error in onComplete and still the intent and Toast may be executed. After committing, squash your commits.

Reply:
Review @sonusourav